.github/workflows/main.yml

```diff
@@ -89,7 +88,6 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ env.RUBY_VERSION }}
```

Comment:
We no longer need to pass this version as the action reads it from the `.ruby-version` file.